fdmi/plugins/sched.c

```diff
@@ -235,77 +254,49 @@ M0_INTERNAL void m0_save_m0_fol_rec(struct m0_fol_rec *rec, const char *prefix)
 	close(fd);
 	M0_LEAVE("fol rec ptr=%p\n", rec);
 }
-*/
-
-#define MAX_LEN 8192
+#endif
 
-char buffer[MAX_LEN];
 M0_INTERNAL char *to_hex(void *addr, int len)
 {
-	int i;
-	int j;
+	int i, j;
 	if ((2 * len) > MAX_LEN) {
-		m0_console_printf( "to_hex() failed with (2 * len) > MAX_LEN\n");
+		fprintf(stderr, "to_hex() failed with (2 * len) > MAX_LEN\n");
 		return NULL;
 	}
-	for(i = 0, j = 0; i < (2 * len) && j < len; ++j) {
+	for(i = 0, j = 0; i < (2 * len) && j < len; ++j)
 		i += sprintf(buffer + i, "%02x", ((char *)addr)[j]);
-	}
 	buffer[2 * len - 2] = '\0';
 	return buffer;
 }
-/*
-const char hexmap[] = {'0', '1', '2', '3', '4', '5', '6', '7',
-                           '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'};
-char *to_hex(char *addr, int len)
-{
-	int i;
-    char *buffer = (char *) malloc(2 * len);
-	if (buffer == NULL) {
-		m0_console_printf( "malloc failed\n");
-		return NULL;
-	}
-    //bzero(buffer, 2*len);
-	for (i = 0; i < len; ++i) {
-		buffer[2 * i]     = hexmap[(addr[i] & 0xF0) >> 4];
-		buffer[2 * i + 1] = hexmap[addr[i] & 0x0F];
-	}
-	buffer[2 * len - 2] = '\0';
-	m0_console_printf(" 2 * i: %d len: %d\n", 2 * i, len);
-	return buffer;
-}
-*/
 
 M0_INTERNAL void m0_dump_m0_fol_rec_to_json(struct m0_fol_rec *rec)
 {
+	struct m0_fol_frag *frag;
 	int i;
 
-	//m0_fol_frag:rp_link to this list
-	struct m0_fol_frag     *frag;
 	m0_tl_for(m0_rec_frag, &rec->fr_frags, frag) {
-
 		struct m0_fop_fol_frag *fp_frag = frag->rp_data;
 		struct m0_cas_op *cas_op = fp_frag->ffrp_fop;
 		struct m0_cas_recv cg_rec = cas_op->cg_rec;
 		struct m0_cas_rec *cr_rec = cg_rec.cr_rec;
 
-		/*
-		m0_console_printf("cas op opcode: %d\n", cas_op->cg_opcode);
-		*/
 		for (i = 0; i < cg_rec.cr_nr; i++) {
 			int len = 0;
 
 			m0_console_printf("{ ");
 
 			len = sizeof(struct m0_fid);
-			m0_console_printf("\"fid\": \"%s\", ", to_hex((void *)&cas_op->cg_id.ci_fid, len));
+			m0_console_printf("\"fid\": \"%s\", ",
```

Comment:
I see m0_console_printf() is replaced with printf().
Why not this?

Reply:
This is sending data to the python path, let's keep it as is. The other places are just messages and errors not needed for functioning.

Reply:
OK